client/src/main/java/org/kakara/client/KakaraGame.java

```diff
@@ -67,6 +68,11 @@ public void start(GameHandler gameHandler) throws Exception {
         gameHandler.getSceneManager().setScene(mainMenuScene);
```

Comment:
Line 67 and 68 should not be there. They were replaced by the first scene method.